index.js

```diff
@@ -0,0 +1,9 @@
+
+const { parse, valid } = require('./lib/input');
+const { execute } = require('./lib/notes');
+
+const action = parse(process.argv);
+const isValid = valid(action); 
+
+if(isValid);
+else console.log('invalid command');
```

Comment:
files should end with a newline. you can configure vs code to add it for you automatically